agency/README.md

```diff
@@ -365,20 +365,21 @@ The reservation method enum describes the different ways in which a passenger ca
+| trip_id                   | UUID          | Required               | UUID for the trip this payload pertains to                                                                                                                                                                                                                                                                                    |
+| provider_id               | UUID          | Required               | Provider which managed this trip                                                                                                                                                                                                                                                                                              |
+| reservation_method        | Enum          | Required               | Way the customer created their reservation, see [reservation-method](#reservation-method)                                                                                                                                                                                                                                     |
+| reservation_time          | Timestamp     | Required               | Time the customer *requested* a reservation                                                                                                                                                                                                                                                                                   |
+| reservation_type          | Enum          | Required               | Type of reservation, see [reservation-type](#reservation-type)                                                                                                                                                                                                                                                                |
+| dispatch_time             | Timestamp     | Conditionally Required | Time the vehicle was dispatched to the customer (required if trip was dispatched)                                                                                                                                                                                                                                             |
+| scheduled_trip_start_time | Timestamp     | Conditionally Required | Time the trip was scheduled to start. Only applicable in the case where `reservation_type` is `scheduled`                                                                                                                                                                                                                     |
```

Comment:
TBD -- could we want to require this in _all_ cases, not just for scheduled reservations? Is there ever a case where there's a reservation and there's no (at least estimated) pickup time mentioned to the customer?